DataManagementSystem/Service/TornadoFileCatalogHandler.py

```diff
@@ -0,0 +1,640 @@
+########################################################################
+# File: FileCatalogHandler.py
+########################################################################
+"""
+:mod: FileCatalogHandler
+
+.. module: FileCatalogHandler
+
+:synopsis: FileCatalogHandler is a simple Replica and Metadata Catalog service
+
+"""
+
```

Comment:
You're missing `__future__` imports here (and possibly in other files).

Reply:
Indeed, I will fix it thanks